versioned_docs/version-1.1.x/surrealql/operators.mdx

```diff
@@ -538,14 +538,14 @@ SELECT * FROM 20 ** 3;
 
 <br />
 
-## `**` {#pow}
+## `%` {#mod}
```

Comment:
@Odonno did you mean to take`**` out?   `%` isn't supported in 1.1.0
<img width="1249" alt="Screenshot 2024-01-10 at 12 47 08" src="https://github.com/surrealdb/docs.surrealdb.com/assets/35943047/c93a8aba-a3fa-43f2-8a8b-e6368bccb2cb">

Reply:
I think this is a mistake on our end. Will remove this and merge

Reply:
I don't understand. Modulo is a new feature, so it's not in 1.0.x but it is available in 1.1.x and later (like nightly), right?